sydent/pipeline.yml

```diff
@@ -5,3 +5,18 @@ steps:
     plugins:
       - docker#v3.0.1:
           image: "matrixdotorg/sydent-build"
+  - command:
+      - "echo '--- Install dependencies'"
+      - "pip install -e ."
+      - "echo '--- Install matrix-is-tester'"
+      - "python -m pip install git+https://github.com/matrix-org/matrix-is-tester.git"
+      - "echo '+++ Run Tests'"
+      - "PYTHONPATH=`pwd` trial matrix-is-tester"
```

Comment:
what's the point of setting PYTHONPATH if we've already pip installed it?

Reply:
mm, good point